src/k8s/objects/v2v/vmware/conversionPod.js

```diff
@@ -0,0 +1,50 @@
+import { PodModel } from '../../../../models';
+import { CONVERSION_BASE_NAME, CONVERSION_GENERATE_NAME } from '../../../requests/v2v';
+
+export const buildConversionPod = ({ volumes, volumeMounts, namespace, serviceAccountName, secretName }) => ({
+  apiVersion: PodModel.apiVersion,
+  kind: PodModel.kind,
+  metadata: {
+    generateName: CONVERSION_GENERATE_NAME,
+    namespace,
+  },
+  spec: {
+    serviceAccountName,
+    containers: [
+      {
+        name: CONVERSION_BASE_NAME,
+        image: 'quay.io/nyoxi/kubevirt-v2v-conversion:1.12.1-1-gf665c0a',
```

Comment:
Please add TODO here for better visibility.
It's a huge gap which need to be resolved together with the v2v-vmware repo

Reply:
done